clang/runtime/dpct-rt/include/dpct/image.hpp

```diff
@@ -840,6 +840,658 @@ static inline image_wrapper_base *create_image_wrapper(image_data data,
   return nullptr;
 }
 
+#ifdef SYCL_EXT_ONEAPI_BINDLESS_IMAGES
```

Comment:
Please put the new added helper code  bindless_images.hpp, and using experimental

Reply:
fixed